src/test/java/org/varroa/remote/codec/decoder/DecodeUtilsTest.java

```diff
@@ -1,20 +1,33 @@
 package org.varroa.remote.codec.decoder;
 
+import com.google.common.base.Utf8;
 import io.netty.buffer.CompositeByteBuf;
 import io.netty.buffer.UnpooledByteBufAllocator;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 import org.varroa.remote.codec.encoder.EncodeUtils;
 
 import static org.junit.Assert.assertEquals;
 
 public class DecodeUtilsTest {
 
+    private final String wabern = "wabern";
+    private final int expectedLength = Utf8.encodedLength(wabern);
+    private CompositeByteBuf buffi;
+
+    @Before
+    public void setUp() {
+        buffi = UnpooledByteBufAllocator.DEFAULT.compositeDirectBuffer(expectedLength);
```

Comment:
Why do you use a compositeDirectBuffer?
```suggestion
        buffi = Unpooled.buffer();
```

Reply:
resolved